ban_appeals/ban_appeals.py

```diff
@@ -133,13 +103,75 @@ async def on_member_join(self, member: discord.Member) -> None:
             # Kick them if they are not banned and not part of the bypass list
             await self._maybe_kick_user(member)
 
+    @checks.has_permissions(PermissionLevel.SUPPORTER)
+    @commands.group(invoke_without_command=True, aliases=("appeal_category",))
+    async def appeal_category_management(self, ctx: commands.Context) -> None:
+        """Group of commands for managing appeal categories."""
+        await ctx.send_help(ctx.command)
+
+    @checks.has_permissions(PermissionLevel.SUPPORTER)
+    @appeal_category_management.command(name="get")
+    async def get_categories(self, ctx: commands.Context) -> None:
+        """Get the list of appeal categories of commands for managing appeal categories."""
+        category_str = ", ".join(map(str, self.appeal_categories)) if self.appeal_categories else "None"
+
+        await ctx.send(f"Currently configured appeal categories are: {category_str}")
```

Comment:
```suggestion
    @checks.has_permissions(PermissionLevel.SUPPORTER)
    @appeal_category_management.command(name="get")
    async def get_categories(self, ctx: commands.Context) -> None:
        """Get the list of appeal categories of commands for managing appeal categories."""
        await self.init_task
        category_str = ", ".join(map(str, self.appeal_categories)) if self.appeal_categories else "None"

        await ctx.send(f"Currently configured appeal categories are: {category_str}")
```

We should be awaiting the init task to not get "None" when the appeal categories haven't been loaded.